app/root.tsx

```diff
@@ -15,6 +15,10 @@ export default function App() {
 			<head>
 				<meta charSet="utf-8" />
 				<meta name="viewport" content="width=device-width, initial-scale=1" />
+        <link
+            rel="stylesheet"
+            href="https://fonts.googleapis.com/css2?family=Plus+Jakarta+Sans:wght@200..700&display=swap"
```

Comment:
The most ideal option for us at PSE is usually to self-host the fonts. (download them and add them to our repo). 

However, we tend to use [Fontsource](https://fontsource.org/) or [Bunny Fonts](https://fonts.bunny.net/) if we're going to go the CDN route as they respect privacy more than Google! (or so they claim)